### PR TITLE
[WebInterface] Add read only display for profile strategy

### PR DESCRIPTION
**Pull request by Herklos**
image

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
👍 
Great ! 
I thinks we could even merge the strategy display in some way in the trading mode, to make it less separate now that it's a read only display

**Comment by Herklos** [comment]
> Great !
> I thinks we could even merge the strategy display in some way in the trading mode, to make it less separate now that it's a read only display

You mean in the same card ?

**Comment by Herklos** [comment]
Screenshot from 2021-04-24 12-24-54
Screenshot from 2021-04-24 12-25-06
Screenshot from 2021-04-24 12-32-17

**Comment by Herklos** [comment]
Screenshot from 2021-04-24 12-35-08

File: Services/Interfaces/web_interface/templates/components/config/profile_tab.html [hunk 1]
```diff
@@ -216,6 +215,11 @@ <h5>
                 {% endif %}
             </div>
         </div>
+        <div class="card-footer">
+            {% if profile.read_only %}
+                <i class="fa fa-info"></i> This profile is ready only, press duplicate to create a copy to be able to edit it.
```
Comment (GuillaumeDSM) [inline Services/Interfaces/web_interface/templates/components/config/profile_tab.html:220]: by puttin this inside a <p> I think we will get a small padding that will make it look nicer

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**